src/coreclr/src/pal/src/libunwind/src/CMakeLists.txt

```diff
@@ -14,8 +14,6 @@ add_definitions(-DPACKAGE_STRING="")
 add_definitions(-DPACKAGE_BUGREPORT="")
 
 add_definitions(-D_GNU_SOURCE)
-# Ensure that the remote and local unwind code can reside in the same binary without name clashing
-add_definitions("-Ddwarf_search_unwind_table_int=UNW_OBJ(dwarf_search_unwind_table_int)")
```

Comment:
Based on https://github.com/libunwind/libunwind/issues/176, this will fix musl arm64 build:

```cmake
if (CLR_CMAKE_TARGET_ALPINE_LINUX AND CLR_CMAKE_TARGET_ARCH_ARM64)
  add_definitions("-D__sigset_t=sigset_t")
endif()
```